.travis.yml

```diff
@@ -23,5 +23,7 @@ script:
 - lintian -i -I --profile debian --show-overrides ../digimend-dkms_*.changes
 - sudo dpkg -i ../digimend-dkms_*_all.deb
 - sudo dpkg -r digimend-dkms
+# Checks build tarball
```

Comment:
Thank you for adding comments, Hai :) However, it's a bit out-of-place alone here. The operation is rather obvious compared to all others, so we better either not have this comment, or document all other operation which are at least this non-obvious, which would be most of them.

Reply:
Also, the "mood" of the commit subject is great, but English and details could use improvement. E.g. `Add tarball build check to Travis CI`.